testing/playwright/page-objects/CreatePlanWizard/CreatePlanWizardPage.ts

```diff
@@ -0,0 +1,50 @@
+import { expect, type Page } from '@playwright/test';
+
+import { GeneralInformationStep } from './steps/GeneralInformationStep';
+import { NetworkMapStep } from './steps/NetworkMapStep';
+import { ReviewStep } from './steps/ReviewStep';
+import { StorageMapStep } from './steps/StorageMapStep';
+import { VirtualMachinesStep } from './steps/VirtualMachinesStep';
+
+export class CreatePlanWizardPage {
+  public readonly generalInformation: GeneralInformationStep;
+  public readonly networkMap: NetworkMapStep;
+  protected readonly page: Page;
+  public readonly review: ReviewStep;
+  public readonly storageMap: StorageMapStep;
+  public readonly virtualMachines: VirtualMachinesStep;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.generalInformation = new GeneralInformationStep(page);
+    this.virtualMachines = new VirtualMachinesStep(page);
+    this.networkMap = new NetworkMapStep(page);
+    this.storageMap = new StorageMapStep(page);
+    this.review = new ReviewStep(page);
+  }
+
+  async clickBack() {
+    await this.page.getByTestId('wizard-back-button').click();
+  }
+
+  async clickNext() {
+    await this.page.getByTestId('wizard-next-button').click();
+  }
+
+  async clickSkipToReview() {
+    await this.page.getByTestId('wizard-review-button').click();
+  }
+
+  async verifyCurrentStep(stepNumber: number) {
+    await expect(this.page.getByTestId(`wizard-step-${stepNumber}`)).toHaveClass(/current/);
+  }
+
+  async verifyPlanCreated(planName: string) {
+    await expect(this.page.getByText(`Plan "${planName}" created successfully`)).toBeVisible();
```

Comment:
I'd rather to not rely on text, as tomorrow someone can decide that this text should be `"${planName}" created successfully` or something else